custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -279,6 +309,15 @@ func getNodeNames(list *v1.NodeList) []string {
 	return resourceNames
 }
 
+func isAlloedReducer(reducer string) bool {
```

Comment:
nit: "isAllowedReducer".

Also, WDYT about changing allowedReducers type to map[string]bool, so that you can just use:

if allowedReducers[reducer] {
  ...
}

Reply:
changed to `map[string]bool` per your suggestion.